### ISSUE TITLE
Handling a deque with maxlen

### ISSUE THREAD

**Issue by sirpercival** [jsonpickle/jsonpickle#121]
Hi all,

jsonpickle doesn't play very nicely with deque if you include a maxlen parameter. I tried to use the fix shown [here on stackoverflow](http://stackoverflow.com/questions/19494734/jsonpickle-ignores-maxlen-of-collection-deque):
```python
class DequeHandler(jsonpickle.handlers.BaseHandler):
    def flatten(self, obj, data):
        return list(obj),obj.maxlen
    def restore(self,obj):
        return deque(obj[0], maxlen=obj[1])
jsonpickle.handlers.register(deque, DequeHandler)
```
 but now the deque is being decoded into a list: [[data], maxlen]. Any suggestions for how to improve the handler? And maybe it might be nice to include something like this in handlers.py?

Thanks!

**Comment by sirpercival** [comment]
Actually, SimpleReduceHandler does perfectly well with deques. I'll do a pull request.

**Comment by davvid** [comment]
Thanks for the fix, merged in da2dd46d3540a6c1c1da83c2ab8b300882d3f769
